libjolie/src/main/java/jolie/lang/KeywordClass.java

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
Rename this class to `Keywords`, get rid of `Constants.Keywords` (move `DEFAULT_HANDLER_NAME` to `Constants`.